### PR TITLE
fix: added SRE fixes

### PR DESCRIPTION
**Pull request by vkalta**
version bump:
@contentstack/cli-audit: 1.3.0 -> 1.3.1

### CODE REVIEW

**Review by netrajpatel — DISMISSED** [review]

**Comment by shafeeqd959** [comment]
@vkalta , only merge this code to dev once we get approval from @AniketDev7

**Comment by antonyagustine** [comment]
@cs-raj Please update the PR with the base branch

**Comment by cs-raj** [comment]
> @cs-raj Please update the PR with the base branch

Done @antonyagustine

**Review by antonyagustine — DISMISSED** [review]

**Review by aman19K — DISMISSED** [review]

File: packages/contentstack-utilities/src/auth-handler.ts
Comment (antonyagustine) [inline packages/contentstack-utilities/src/auth-handler.ts:120]: We can't use HTTPS as we creating the local server, To create an https server on nodejs we need a system footprint (Key and certificate files) Which we can't get access to from the user's system. So we need to revert these changes.

cc @netrajpatel @cs-raj

**Review by aman19K — APPROVED** [review]

**Review by cs-raj — APPROVED** [review]

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**
